src/cozy/wrappers/react-wrapper.jsx

```diff
@@ -42,15 +65,7 @@ const ReactWrapper = ({ client, bitwardenData, ...props }) => {
           require(`../react/locales/${appLocale}.json`)
         }
       >
-        <CozyProvider client={client}>
-          <VaultProvider instance={client.getStackClient().uri}>
-            <BitwardenSettingsContext.Provider value={bitwardenData}>
-              <BreakpointsProvider>
-                <HashRouter>{props.children}</HashRouter>
-              </BreakpointsProvider>
-            </BitwardenSettingsContext.Provider>
-          </VaultProvider>
-        </CozyProvider>
+        <CozyProvider client={client}>{subComponent}</CozyProvider>

```

Comment:
check if Reverted later

Reply:
I don't understand this comment. What do you mean?

Reply:
Ahh, do you refer to the commit message saying `This is a temporary fix that may be reverted in the future' ?
Forgot that one. I can revert back, I just need to move the `vaultData` part in `AngularWrapperComponent`.

Reply:
Exactly what I was thinking! LEt's move the vaultData part ;)

Reply:
Done: https://github.com/cozy/cozy-pass-web/pull/9/commits/8a3e24b40916d1e707d2b09de5058c4edc8e8cbd